### PR TITLE
FFM-4409: updates Aug 24 relnotes

### PR DESCRIPTION
**Pull request by StrawberryPad**
This has already been tech-reviewed/approved offline by @conormurray95

### CODE REVIEW

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -664,6 +666,12 @@ The .NET SDK has been updated to version 1.1.3. Fixes in this update include:
 - When evaluating Target Groups that used an IN operator, the SDK was only evaluating the first Target. (FFM-4358)
   The logic has now been fixed so that the SDK will check all values when an IN operator is used for a Target Group.
 
+#### Relay proxy
+
+* The proxy had a dependancy on a jwt package that is no longer maintained. This fix updated the jwt dependency to a package that is maintained. (FFM-3867)
+* The proxy had a dependency on ff-server, which is in a private repo. This fix removed the dependency on ff-server. (FFM-3965)
```
Comment (SmritiSatya) [inline release-notes/feature-flags.md:672]: Can we say repository instead of repo?

**Review by SmritiSatya — APPROVED** [review]

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -664,6 +666,12 @@ The .NET SDK has been updated to version 1.1.3. Fixes in this update include:
 - When evaluating Target Groups that used an IN operator, the SDK was only evaluating the first Target. (FFM-4358)
   The logic has now been fixed so that the SDK will check all values when an IN operator is used for a Target Group.
 
+#### Relay proxy
+
+* The proxy had a dependancy on a JWT package that is no longer maintained. This fix updated the JWT dependency to a package that is maintained. (FFM-3867)
```
Comment (aimurphy) [inline release-notes/feature-flags.md:671]:
```suggestion
* The proxy had a dependency on a JWT package that is no longer maintained. This fix updated the JWT dependency to a package that is maintained. (FFM-3867)
```

File: release-notes/feature-flags.md [hunk 3]
```diff
@@ -664,6 +666,12 @@ The .NET SDK has been updated to version 1.1.3. Fixes in this update include:
...
+* The proxy had a dependency on ff-server, which is in a private repository. This fix removed the dependency on ff-server. (FFM-3965)
+* Harness provided a tool to generate off-line config files. For details, go to [Run the Relay Proxy in offline mode](/docs/feature-flags/ff-using-flags/relay-proxy/deploy-relay-proxy#run-the-relay-proxy-in-offline-mode) (FFM-3772)
```
Comment (aimurphy) [inline release-notes/feature-flags.md:673]:
```suggestion
* Harness provided a tool to generate offline config files. For details, go to [Run the Relay Proxy in offline mode](/docs/feature-flags/ff-using-flags/relay-proxy/deploy-relay-proxy#run-the-relay-proxy-in-offline-mode) (FFM-3772)
```

Should it be off-line or offline, like in the link text?

**Review by aimurphy — APPROVED** [review]
approved w 2 sugs

**Pull request merged**